### DISCUSSION TITLE
values needed to show the elevator trim?

### DISCUSSION THREAD

**Discussion by johans1234** [nguyenquyhy/Flight-Tracker-StreamDeck#158] [General]
I would like to create a gauge which shows me the position of my elevator trim. (as shown in your example)
Which values do I have to put into the fields?

**Comment by rmroc451** [comment]
howdy! you can grab the example profile off the PR:

https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/pull/58 -> https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/files/5577145/17-Examples.and.icons.you.can.distribute.zip

**Reply by johans1234** [reply]
OK, think that will help and answer my questions - many thanks

**Reply by johans1234** [reply]
are there more example profiles? Sorry, I am not very familiar with git and I don`t know, how to navigate to the repository or folder, where the file you have linked is stored. Maybe there are some other valueable things and maybe I can upload my own profile, which I use for the Carenado M20R Ovation.

**Reply by nguyenquyhy** [reply]
@johans1234 we have a thread where everyone can share their profile. https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/discussions/128

**Comment by Nokeel** [comment]
Hi!
I also am trying to set up trim and flaps indicators, I tried manually, gauges remained unresponsive to sim, no changes when trimming.
So I looked up here and loaded the example, but when loading the example profile, this is what I get 
![trimDeck](https://user-images.githubusercontent.com/91840301/135762058-86ed51b9-3b22-4546-b140-dbad0f8857c9.jpg)

Thanks for any help provided

**Reply by Nokeel** [reply]
nvm, I had to change  ELEVATOR_TRIM_PCT to ELEVATOR TRIM PCT    :)

**Reply by nguyenquyhy** [reply]
Those 2 should both work. I would guess that the problem is with a trailing space.
